### PR TITLE
[FLINK-22518][docs] Translate the page of "High Availability (HA)" in…

### PR DESCRIPTION
**Pull request by movesan**
## What is the purpose of the change

Translate "High Availability" page into Chinese.

The model of "High Availability (HA)" contains the following three pages: 
https://ci.apache.org/projects/flink/flink-docs-release-1.12/zh/deployment/ha
https://ci.apache.org/projects/flink/flink-docs-release-1.12/zh/deployment/ha/zookeeper_ha.html
https://ci.apache.org/projects/flink/flink-docs-release-1.12/zh/deployment/ha/kubernetes_ha.html

## Brief change log

  - *translate flink/docs/content.zh/docs/deployment/ha/_index.md*
  - *translate flink/docs/content.zh/docs/deployment/ha/overview.md*
  - *translate flink/docs/content.zh/docs/deployment/ha/zookeeper_ha.md*
  - *translate flink/docs/content.zh/docs/deployment/ha/kubernetes_ha.md*

## Verifying this change

This change is a trivial rework / code cleanup without any test coverage.

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): (no)
  - The public API, i.e., is any changed class annotated with `@Public(Evolving)`: (no)
  - The serializers: (no )
  - The runtime per-record code paths (performance sensitive): (no)
  - Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Kubernetes/Yarn/Mesos, ZooKeeper: (no)
  - The S3 file system connector: (no)

## Documentation

  - Does this pull request introduce a new feature? (no)
  - If yes, how is the feature documented? (no)

### CODE REVIEW

File: docs/content.zh/docs/deployment/ha/kubernetes_ha.md [hunk 1]
```diff
@@ -24,52 +24,50 @@ specific language governing permissions and limitations
...
 
-- [high-availability]({{< ref "docs/deployment/config" >}}#high-availability-1) (required): 
-The `high-availability` option has to be set to `KubernetesHaServicesFactory`.
+- [high-availability]({{< ref "docs/deployment/config" >}}#high-availability-1) (必要的):
+`high-availability` 选项必须设置为 `KubernetesHaServicesFactory`.
 
-```yaml
-high-availability: org.apache.flink.kubernetes.highavailability.KubernetesHaServicesFactory
-```
+  ```yaml
```
Comment (Myasuka) [inline docs/content.zh/docs/deployment/ha/kubernetes_ha.md:48]: These spaces are not necessary, and you could also check all additional spaces below.

File: docs/content.zh/docs/deployment/ha/kubernetes_ha.md [hunk 2]
```diff
@@ -24,52 +24,50 @@ specific language governing permissions and limitations
...
-## Prerequisites
+## 准备
 
-In order to use Flink's Kubernetes HA services you must fulfill the following prerequisites:
+为了使用 Flink 的 Kubernetes 高可用服务，您必须满足以下先决条件:
 
 - Kubernetes >= 1.9.
-- Service account with permissions to create, edit, delete ConfigMaps.
-  Take a look at how to configure a service account for [Flink's native Kubernetes integration]({{< ref "docs/deployment/resource-providers/native_kubernetes" >}}#rbac) and [standalone Flink on Kubernetes]({{< ref "docs/deployment/resource-providers/standalone/kubernetes" >}}#kubernetes-high-availability-services) for more information.
+- 具有创建、编辑、删除 ConfigMaps 权限的服务帐户。想了解更多信息，请查看如何使用 [原生的 Kubernetes 集成]({{< ref "docs/deployment/resource-providers/native_kubernetes" >}}#rbac) 和 [在 Kubernetes 上独立部署 Flink]({{< ref "docs/deployment/resource-providers/standalone/kubernetes" >}}#kubernetes-high-availability-services) 配置服务帐户。
```
Comment (Myasuka) [inline docs/content.zh/docs/deployment/ha/kubernetes_ha.md:38]: `standalone` 翻译成 `独立`似乎不妥

File: docs/content.zh/docs/deployment/ha/overview.md [hunk 3]
```diff
@@ -26,56 +26,50 @@ specific language governing permissions and limitations
...
 
 {{< top >}}
 
-## High Availability data lifecycle
+## 高可用数据生命周期
 
-In order to recover submitted jobs, Flink persists metadata and the job artifacts.
-The HA data will be kept until the respective job either succeeds, is cancelled or fails terminally.
-Once this happens, all the HA data, including the metadata stored in the HA services, will be deleted.  
+为了恢复提交的作业，Flink 持久化元数据和 job 组件。高可用（HA）数据将一直保存，直到相应的作业执行成功、被取消或最终失败。一旦发生这种情况，将删除所有高可用（HA）数据，包括存储在高可用（HA）服务中的元数据。
```
Comment (Myasuka) [inline docs/content.zh/docs/deployment/ha/overview.md:73]: 需要每次翻译 `高可用`时，都需要带上 `HA`么？

File: docs/content.zh/docs/deployment/ha/zookeeper_ha.md [hunk 4]
```diff
@@ -24,113 +24,104 @@ specific language governing permissions and limitations
...
 ```bash
 server.X=addressX:peerPort:leaderPort
 [...]
 server.Y=addressY:peerPort:leaderPort
 ```
 
-The script `bin/start-zookeeper-quorum.sh` will start a ZooKeeper server on each of the configured hosts. 
-The started processes start ZooKeeper servers via a Flink wrapper, which reads the configuration from `conf/zoo.cfg` and makes sure to set some required configuration values for convenience. 
-In production setups, it is recommended to manage your own ZooKeeper installation.
+脚本 `bin/start-zookeeper-quorum.sh` 将在每个配置的主机上启动一个 ZooKeeper 服务器。该进程是通过 Flink 包装器来启动的 ZooKeeper 服务器，该包装器从 `conf/zoo.cfg` 读取配置，并确保设置一些必要的配置值以方便使用。在生产设置中，建议您管理自己的 ZooKeeper 安装。
```
Comment (Myasuka) [inline docs/content.zh/docs/deployment/ha/zookeeper_ha.md:125]: 建议最后一句换行，可能更贴切的描述是`在生产环境中，建议您自行管理Zookepper的安装与部署`

**Review by Myasuka — COMMENTED** [review]
Thanks for your contributions!
I have given some suggestions below, please take a look.

File: docs/content.zh/docs/deployment/ha/kubernetes_ha.md (hunk 1 above)
Comment (movesan) [reply docs/content.zh/docs/deployment/ha/kubernetes_ha.md]: 你好，感谢你的建议，之前是考虑和 docs/content.zh/docs/deployment/ha/zookeeper_ha.md 中的缩进保持一致。现已去掉多余缩进，与原版本保持一致。

File: docs/content.zh/docs/deployment/ha/kubernetes_ha.md (hunk 2 above)
Comment (movesan) [reply docs/content.zh/docs/deployment/ha/kubernetes_ha.md]: 你好，感谢指正，目前已修改为 `单节点`

File: docs/content.zh/docs/deployment/ha/overview.md (hunk 3 above)
Comment (movesan) [reply docs/content.zh/docs/deployment/ha/overview.md]: 多谢指正，已修改。

File: docs/content.zh/docs/deployment/ha/zookeeper_ha.md (hunk 4 above)
Comment (movesan) [reply docs/content.zh/docs/deployment/ha/zookeeper_ha.md]: 好建议

**Review by Myasuka — CHANGES_REQUESTED** [review]
@movesan, Overall, it looks good to me. Would you please combine the three commits into one single commit and ensure the commit message is full and clean? You can then force push your commits to your branch to update this PR. @

**Comment by movesan** [comment]
> @movesan, Overall, it looks good to me. Would you please combine the three commits into one single commit and ensure the commit message is full and clean? You can then force push your commits to your branch to update this PR.
@Myasuka Hi Myasuka, thank you for your reply. I have combined the three commits into one single commit, Could you merge this PR into the master? Thanks a lot.

**Comment by Myasuka** [comment]
@movesan , thanks for your update!
However, the commit message is still not full and clean, it should not be `[FLINK-22518][docs] Translate the page of "High Availability (HA)" in…`. Please amend your commit message and then force push it.

**Comment by movesan** [comment]
@Myasuka Hi Myasuka, thank you for your review, I have modified to a more detailed comment. Please check it, thx.

**Review by Myasuka — APPROVED** [review]
LGTM

**Pull request merged**